oasislmf/cmd/model.py

```diff
@@ -212,7 +212,7 @@ def action(self, args):
 
         self.logger.info('Generating a canonical {} file {} from source {} file {}'.format(_sft, output_file_path, _sft, source_file_path))
 
-        translator = Translator(source_file_path, output_file_path, xsd_validation_file_path, xslt_transformation_file_path, append_row_nums=True)
+        translator = Translator(source_file_path, output_file_path, xslt_transformation_file_path, xsd_validation_file_path, append_row_nums=True)
```

Comment:
Looks like the only difference here is you've swapped the order of the transformation and validation file path arguments?